webpack/scenes/ContentViews/Create/CreateContentViewModal.js

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import { Modal, Button, ModalVariant } from '@patternfly/react-core';
+import ContentViewForm from './CreateContentViewForm';
+
+const CreateContentViewModal = ({ show }) => {
+  const [modalOpen, setModalOpen] = useState(show);
+  useEffect(
+    () => {
+      setModalOpen(show);
+    },
+    [show]
+  );
+   return (
+      <Fragment>
```

Comment:
This was was wrong in my example :flushed: - you can drop the `Fragment` wrapping tags as its one component